src/mappings/clearingHouse.ts

```diff
@@ -339,6 +348,11 @@ export function handleLiquidityChanged(event: LiquidityChangedEvent): void {
     market.baseAmount = market.baseAmount.plus(liquidityChanged.base)
     market.quoteAmount = market.quoteAmount.plus(liquidityChanged.quote)
 
+    const diff = abs(traderMarket.takerPositionSize).minus(abs(beforePositionSize))
```

Comment:
prefer to move this into the hard fix part (after line 321) and remove the "if (diff > 0)", since updating open interest here is for hard fix only. We can move line 347 - 349 up.